src/test/resources/contracts/offenders.avsc

```diff
@@ -0,0 +1,465 @@
+{
+  "type" : "record",
+  "namespace" : "prisons",
+  "service": "nomis",
+  "version": "1.0.0",
+  "name" : "OFFENDERS",
+  "description": "A person who is recorded as being convicted of a criminal activity or held on remand awaiting judicial process in order that they may be tracked within the Prison and / or Probation services. This may be as a remand prisoner, a prisoner, a person subject to probation supervision. ISSUE: Potential duplication of attribute values on occurrences which are recording aliases of the offender.",
+  "contacts" : {
+    "author": "dominic.messenger@digital.justice.gov.uk",
+    "team": "https://dsdmoj.atlassian.net/wiki/spaces/DPR/overview?homepageId=4015489077",
+    "slack": "#ask_dpr"
+  },
+  "database": {
+    "type": "oracle",
+    "version": "12",
+    "name": "CNOMT3"
+  },
+  "propagation-policy": "1.day",
+  "retention-policy": "999.years",
+  "fields" : [
+    {
+      "name": "OFFENDER_ID",
```

Comment:
Just a general note. Where we've declared a default value in the schema that defines our format, you can omit the value. In this case this would apply to
* `sensitivity` - default value declared as `public`
* `certified` - default value declared as `false`

Reply:
I will leave it to @dominic-messenger-moj to comment on this since the wiki page says the default value for the above. Happy to remove if we all agreed in a tiny PR. Merging this now

Reply:
Propagation Policy should be empty (by default propagation is incremental)
Retention Policy should be blank (by default this is the departmental policy)

Reply:
Sensitivty and Certified can be removed for all fields.

Reply:
Also remove the glossary-index for now for all fields except for offender:id. The remainder we don't actually know what they mean and we shouldn't make any assumptions.